### PR TITLE
Events page design

### PR DESCRIPTION
**Pull request by YaronMiro**
#87
wip: porting design from #96

### CODE REVIEW

**Comment by YaronMiro** [comment]
@amitaibu This PR is ready.
I'm waiting for the tests to pass and then I will squash merge & deploy is it OK by You ?

**Desktop:**
desktop

**Mobile:**
mobile

**My account page list style:**
my account list style

**Comment by amitaibu** [comment]
:+1:

**Comment by amitaibu** [comment]
Merged, thanks.

File: src/assets/scss/_events-page.scss [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    li {
+      position: relative;
+      margin-top: 10px;
+      padding: 5px 9px;
+      border: 1px solid #E2E2E2;
+      border-radius: 5px;
+      transition: background 0.2s ease-in;
+
+      &:first-child {
+        margin-top: 0;s
```
Comment (amitaibu) [inline src/assets/scss/_events-page.scss]: `0;s` ...

**Pull request closed without merging**